src/pageObject.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
-class PageObject {
+class PageObject extends Header {
+  constructor(url) {
+    super();
+    this.url = url;
+  }
+
+  get header() {
```

Comment:
The header property should be initialized in the constructor, not as a getter. 😊

Reply:
Done